A32NX/html_ui/Pages/A32NX_Core/A32NX_Pilots.js

```diff
@@ -0,0 +1,9 @@
+class A32NX_Pilots {
+    init() {
+        const storedPilotVis = parseInt(NXDataStore.get("CONFIG_PILOT_VISIBILITY", "0"));
+        const storedCoPilotVis = parseInt(NXDataStore.get("CONFIG_COPILOT_VISIBILITY", "0"));
```

Comment:
I'm not convinced NXDataStore is available at init time: on first run, this seems to be causing an error in the MCDU, and it also doesn't seem to be actually loading the values anyway (so if I enable the copilot in the options then restart the flight, the copilot model won't be present but the options screen says "SHOW").

Reply:
> I'm not convinced NXDataStore is available at init time: on first run, this seems to be causing an error in the MCDU, and it also doesn't seem to be actually loading the values anyway (so if I enable the copilot in the options then restart the flight, the copilot model won't be present but the options screen says "SHOW").

I'm not JS specialist, and dont know how to do correctly. Do you have a solution?

Reply:
it looks correct i think but there is a error? maybe i can help you because i using that a 1000 times XDDD
i turn on my sim. i wish i can help

Reply:
> it looks correct i think but there is a error? maybe i can help you because i using that a 1000 times XDDD
> i turn on my sim. i wish i can help

I didn't see any errors. Restarting flight plane also work fine, and i see copilots. Same stuf is used in A32NX_BaroSelector.js

Reply:
@mixMugz 
you did a great job!
i have a question. did you using msfs web dev kit? if you add some code in mcdu, core, etc.... if you break your code, other code all not working due to that error. 

i recommend using this - https://github.com/dga711/msfs-webui-devkit

Reply:
I made only first steps in JS. Thanks for recomendation, but i dont know yet how to use it =)

Reply:
> I made only first steps in JS. Thanks for recomendation, but i dont know yet how to use it =)

i mention on discord. using that :)

Reply:
> > I made only first steps in JS. Thanks for recomendation, but i dont know yet how to use it =)
> 
> i mention on discord. using that :)

Thanks!